src/app/pages/connect-page/connection-details/connection-details.page.ts

```diff
@@ -29,7 +31,19 @@ export class ConnectionDetailsPage implements OnInit{
     //       self.navigateTo('/list-connections');
     //     }
     // }
-    
+    onRemoveBtnClick() {
```

Comment:
There should be a blank line between any class level functions, and any comments that do not apply to what follows them (in this case, the commented out method/function (ending at line 33) does not relate to the onRemoveBtnClick() function. Add a blank line.